src/browser/components/molecules/OverdueTaskList/index.js

```diff
@@ -0,0 +1,124 @@
+import React, { Component } from 'react'
+import {fetchURL, toStandardDate } from '../../../common/utils'
+import moment from 'moment'
+import styles from './index.css'
+
+export default class OverdueTaskList extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      tasks:[],
+      users:[]
+    }
+  }
+
+  componentDidMount(){
+    this.fetchTasks()
+  }
+
+  fetchTasks(){
+    const options = {
+      method: 'GET',
+      mode: 'cors',
+      headers: new Headers({
+        'Accept': 'application/json',
+        'Content-Type': 'application/json',
+      }),
+      credentials: 'same-origin'
+    }
+    fetch('api/task/all', options)
```

Comment:
can you use the function in this file - https://github.com/GuildCrafts/newbie/blob/master/src/browser/common/utils.js